app/src/main/java/org/grammatek/simacorrect/spellcheckerservice/YfirlesturAnnotation.kt

```diff
@@ -14,10 +15,14 @@ import java.lang.NullPointerException
  */
 class YfirlesturAnnotation(
     response: YfirlesturResponse?,
-    private val _annotations: List<Annotations> = response?.result?.get(0)?.get(0)?.annotations ?: throw NullPointerException("annotations null"),
-    private val _originalText: String = response?.text ?: throw NullPointerException("original text null")
+    text: String?,
+    _response: YfirlesturResponse? = response,
+    _originalText: String = response?.text ?: throw NullPointerException("original text null"),
+    private val _unalteredOriginalText: String? = text
 ) {
-    val suggestionsIndexes: MutableList<SuggestionIndexes> = mutableListOf()
+    private val _annotations: List<List<Annotations>>
+    val suggestionsIndices: MutableList<AnnotationIndices> = mutableListOf()
```

Comment:
could you describe this variable and the following because these are actually accessed from outside the class ?

Reply:
the following one is accessed within the class, turning it into a private val and adding a comment for `suggestionsIndices`